backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -72,6 +76,15 @@ public TestResultUpload processResultCSV(InputStream csvStream)
     return result;
   }
 
+  public Page<TestResultUpload> getUploadSubmissions(
+      Date startDate, Date endDate, int pageNumber, int pageSize) {
+    Organization org = _orgService.getCurrentOrganization();
+    PageRequest pageRequest =
```

Comment:
This PageRequest is pretty neat, seems like a much nicer way to deal with pagination